Disable flake8 and isort - these are linters, so not very useful for downstream.

--- a/setup.cfg
+++ b/setup.cfg
@@ -61,8 +61,6 @@ install_requires =
 tests_require = 
 	pytest-runner
 	pytest-cov
-	pytest-flake8
-	pytest-isort
 python_requires = >= 3.6
 
 [options.entry_points]
@@ -80,8 +78,6 @@ doc =
 test = 
 	pytest-runner
 	pytest-cov
-	pytest-flake8
-	pytest-isort
 
 [bdist_wheel]
 python-tag = py3
@@ -94,7 +90,6 @@ build-dir = docs/_build
 test = pytest
 
 [tool:pytest]
-addopts = --flake8 --isort
 norecursedirs = build dist .cache .eggs .git
 
 [coverage:run]
